.ci/bears.prune.sh

```diff
@@ -0,0 +1,160 @@
+elif [[ "$BEARS" == "ruby" ]]; then
+  BEARS=gem
+elif [[ "${BEARS/node/}" != "$BEARS" ]]; then
+  BEARS=npm
+elif [[ "$BEARS" == "r" ]]; then
+  BEARS=rscript
+elif [[ "$BEARS" == "generic" ]]; then
+  BEARS=apt-get
+fi
+
+bears=$(find bears -type f -and -name '*Bear.py' | sort)
+
+yield_result_bears=$(grep -m 1 -l 'yield Result' $bears)
+
+non_yield_result_bears=$(comm -23 <(ls $bears) <(ls $yield_result_bears))
+
+requirement_bears=$(grep -m 1 -l 'Requirement(' $bears)
+
+cabal_requirement_bears=$(grep -m 1 -l CabalRequirement $requirement_bears)
+gem_requirement_bears=$(grep -m 1 -l GemRequirement $requirement_bears)
+go_requirement_bears=$(grep -m 1 -l GoRequirement $requirement_bears)
+npm_requirement_bears=$(grep -m 1 -l NpmRequirement $requirement_bears)
+rscript_requirement_bears=$(grep -m 1 -l RscriptRequirement $requirement_bears)
+
+pip_requirement_bears=$(grep -m 1 -l PipRequirement $requirement_bears)
+
+non_pip_runtime_requirement_bears="$cabal_requirement_bears $gem_requirement_bears $go_requirement_bears $npm_requirement_bears $rscript_requirement_bears"
+
+other_requirement_bears=$(comm -23 <(ls $requirement_bears) <(ls $non_pip_runtime_requirement_bears $pip_requirement_bears))
+
+distribution_requirement_bears=$(grep -m 1 -l 'DistributionRequirement(' $requirement_bears)
+
+apt_get_requirement_bears=$(grep -m 1 -l 'apt_get' $distribution_requirement_bears)
+
+# Verify that DistributionRequirement is the only other Requirement subclass used
+unknown_requirement_bears=$(grep -m 1 -Pl '(?<!Distribution)Requirement\(' $other_requirement_bears || true)
+
+if [[ -n "$unknown_requirement_bears" ]]; then
+  echo "Unknown requirements for $unknown_requirement_bears"
+  exit 1
+fi
+
+pip_only_requirement_bears=$(comm -23 <(ls $pip_requirement_bears) <(ls $non_pip_runtime_requirement_bears))
+pip_plus_requirement_bears=$(comm -23 <(ls $pip_requirement_bears) <(ls $pip_only_requirement_bears))
+
+no_requirement_bears=$(comm -23 <(ls $bears) <(ls $requirement_bears))
+
+clang_bears=''
+other_bears=''
+for bear in $no_requirement_bears; do
+  if [[ ${bear/Clang/} != $bear ]]; then
+    clang_bears="$clang_bears $bear"
+  else
+    other_bears="$other_bears $bear"
+  fi
+done
+
+executable_linter_bears=$(grep -m 1 -l '@linter(.' $other_bears)
+executable_other_bears=$(grep -m 1 -l 'run_shell_command' $other_bears)
+
+other_bears=$(comm -23 <(ls $other_bears) <(ls $executable_linter_bears $executable_other_bears))
+
+non_yield_result_other_bears=$(comm -23 <(ls $other_bears) <(ls $yield_result_bears))
+
+if [[ -n "$non_yield_result_other_bears" ]]; then
+  echo "Unknown bear type $non_yield_result_other_bears"
+  exit 1
+fi
+
+echo Other exe $executable_linter_bears $executable_other_bears
+
+python_bears="$pip_only_requirement_bears $clang_bears $other_bears"
+
+non_python_bears=$(comm -23 <(ls $bears) <(ls $python_bears))
+
+cabal_requirement_bears="$cabal_requirement_bears bears/haskell/HaskellLintBear.py bears/shell/ShellCheckBear.py"
```

Comment:
`ShellCheckBear` has a Cabal Requirement now

Ref: https://github.com/coala/coala-bears/pull/1496

Reply:
Also this can needs to be solved: https://github.com/coala/coala-bears/issues/1358 so, that we can remove `HaskellLintBear`